### PR TITLE
Fix macOS builds

### PR DESCRIPTION
**Pull request by sciencewhiz**
Switch to macos-13 runners and install texinfo
macos-latest (14) runners are arm64 only. https://github.com/actions/runner-images/issues/9741 Running the x86_64 build on arm64 gives the following error: `[FATAL]: Currently canadian builds require a x86_64 build system'

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -544,4 +544,4 @@ jobs:
         with:
           allowUpdates: true
           artifacts: "**/*"
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (PeterJohnson) [inline .github/workflows/ci.yml:547]: Unnecessary change (also removing EOL at EOF).

**Review by PeterJohnson — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rzblue) [reply .github/workflows/ci.yml:547]: That's what jinja outputs

**Review by PeterJohnson — APPROVED** [review]
